src/harlequin/components/adapter_details.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
+
+from textual import events
+from textual.app import ComposeResult
+from textual.containers import Vertical, VerticalScroll
+from textual.screen import ModalScreen
+from textual.widgets import Markdown, Static
+
+
+class VerticalSuppressClicks(Vertical):
+    def on_click(self, message: events.Click) -> None:
+        message.stop()
+
+
+class AdapterDetailsScreen(ModalScreen):
```

Comment:
Let's name this screen something like `DebugDetailsScreen` or `InformationScreen`

Reply:
Actually, since you're re-using the HelpScreen, best long-term to refactor the common bits out into a Screen that can be used by both Help and this screen. That way you don't have to repeat the CSS, key handling etc.